paddle/fluid/operators/mkldnn/fc_mkldnn_op.cc

```diff
@@ -68,8 +69,8 @@ class FCPrimitiveFactory {
  private:
   void UpdateDataPointers(const ExecutionContext& ctx, Tensor* out,
                           const Tensor* in) {
-    input_->set_data_handle(const_cast<T*>(in->data<T>()));
-    output_->set_data_handle(out->mutable_data<T>(ctx.GetPlace()));
+    input_->set_data_handle(to_void_cast(in->data<T_in>()));
```

Comment:
I'm removing a const cast here, hence the approval requirement. (It's necessary so that I can replace former `T` type for `T_in` which indicates input type.